Content.Server/Resist/ResistLockerSystem.cs

```diff
@@ -0,0 +1,124 @@
+        }
+
+
+        private void OnRelayMovement(EntityUid uid, ResistLockerComponent component, RelayMovementEntityEvent args)
+        {
+            TryComp(uid, out EntityStorageComponent? storageComponent);
+            if (!Resolve(uid, ref storageComponent))
+                return;
+
+            if (!component.IsResisting)
+            {
+                if (TryComp<LockComponent>(uid, out var lockComponent) && lockComponent.Locked || storageComponent.IsWeldedShut)
+                {
+                    AttemptResist(args.Entity, uid, storageComponent, component);
+                } 
+            }
+
+        }
+        private void AttemptResist(EntityUid user, EntityUid target, EntityStorageComponent? storageComponent, ResistLockerComponent? resistLockerComponent)
+        {
+            if (!Resolve(target, ref storageComponent, ref resistLockerComponent))
+                return;
+
+            resistLockerComponent.CancelToken = new();
+            var doAfterEventArgs = new DoAfterEventArgs(user, resistLockerComponent.ResistTime, resistLockerComponent.CancelToken.Token, target)
+            {
+                BreakOnTargetMove = false,
+                BreakOnUserMove = true,
+                BreakOnDamage = true,
+                BreakOnStun = true,
+                NeedHand = false, //No hands 'cause we be kickin'
+                BroadcastFinishedEvent = new ResistDoAfterComplete(storageComponent, resistLockerComponent, user, target),
+                BroadcastCancelledEvent = new ResistDoAfterCancelled(user, resistLockerComponent)
+            };
+
+            resistLockerComponent.IsResisting = true;
+            _popupSystem.PopupEntity(Loc.GetString("resist-locker-component-start-resisting"), user, Filter.Entities(user));
+            _doAfterSystem.DoAfter(doAfterEventArgs);
+        }
+
+        private void OnDoAfterComplete(ResistDoAfterComplete ev)
+        {
+            ev.ResistComponent.IsResisting = false;
+
+            if (ev.StorageComponent.IsWeldedShut)
+            {
+                ev.StorageComponent.IsWeldedShut = false;
+            }
+
+            if (TryComp<LockComponent>(ev.Target, out var lockComponent))
+            {
+                lockComponent.Locked = false;
+            }
+
+            ev.StorageComponent.TryOpenStorage(ev.User);
+        }
+
+        private void OnDoAfterCancelled(ResistDoAfterCancelled ev)
+        {
+            ev.ResistComponent.IsResisting = false;
+            _popupSystem.PopupEntity(Loc.GetString("resist-locker-component-resist-interrupted"), ev.User, Filter.Entities(ev.User));
+        }
+
+        private void OnRemovedFromContainer(EntityUid uid, ResistLockerComponent component, EntRemovedFromContainerMessage message)
+        {
+            if (component.CancelToken != null)
+            {
+                component.IsResisting = false;
+                component.CancelToken.Cancel();
+            }
+              
+        }
```

Comment:
```suggestion
        }

```

Reply:
There's a lot of weird formatting in the file. Do a check for empty lines and correct formatting. Or just run your IDE's formatter.